Marlin/Marlin_main.cpp

```diff
@@ -3752,6 +3752,9 @@ inline void gcode_G28() {
     #if ENABLED(AUTO_BED_LEVELING_UBL)
       const bool bed_leveling_state_at_entry = ubl.state.active;
     #endif
+	#if ENABLED(MESH_BED_LEVELING)
+	  mbl.set_reactivate(mbl.active());
+	#endif
```

Comment:
We can't accept PRs with tabs. Please use 2 spaces. 
http://marlinfw.org/docs/development/coding_standards.html

So I will this PR and make the patch directly.